Source/Renderer/PickFramebuffer.js

```diff
@@ -114,6 +114,9 @@ define([
 
                 var object = context.getObjectByPickColor(colorScratch);
                 if (defined(object)) {
+                    if (object.id && (typeof object.id === 'object')) {
+                        return (object.id.selectable) ? object : undefined;
```

Comment:
Hey guys, sorry I took so long to respond to this, the email just got lost in the sauce around the holidays.  I would definitely be interested in working on this.  I agree that the `pickPriority` option is preferrable to the `selectable`.  I think I was in a bit of an application-specific mindset there.
I will dig around in the DataSourceDisplay and such to get a better idea of whats going on there while I await instructions.
Again, sorry for taking so long to respond.